tsfc/tensor.py

```diff
@@ -83,7 +83,7 @@ def classify(quadrature_indices, expression):
     return result
 
 
-def flatten(var_reps):
+def flatten(var_reps, index_cache):
     for variable, reps in var_reps:
         expressions = reps
         for expression in expressions:
```

Comment:
I should probably do something more here not to break `EnrichedElement`s in the tensor mode. This was just added because `flatten` not accepting the extra argument fails test cases in TSFC.